views/root.tmpl

```diff
@@ -313,70 +306,36 @@
             </div>
             <div class="col-md-6">
-                <strong>Next Price Window</strong>
+                <strong>Block Subsidy</strong>
                 <table class="table table-sm">
                     <thead>
-                        <th>Min</th>
-                        <th>Max</th>
-                        <th>Guess</th>
+                        <th>Developer</th>
+                        <th>POS</th>
+                        <th>POW</th>
+                        <th>Total</th>
                     </thead>
                     <tbody>
                         <tr>
-                            <td><span id="sdiffmin" >{{printf "%12.2f" .StakeSummary.StakeDiff.Estimates.Min}}</span></td>
-                            <td><span id="sdiffmax" >{{printf "%12.2f" .StakeSummary.StakeDiff.Estimates.Max}}</span></td>
-                            <td><span id="sdiffexp" >{{printf "%12.2f" .StakeSummary.StakeDiff.Estimates.Expected}}</span></td>
+                            <td><span id="bsubsidy_dev" >{{.BlockSummary.NextBlockSubsidy.Developer}}</span></td>
+                            <td><span id="bsubsidy_pos" >{{.BlockSummary.NextBlockSubsidy.PoS}}</span></td>
+                            <td><span id="bsubsidy_pow" >{{.BlockSummary.NextBlockSubsidy.PoW}}</span></td>
+                            <td><span id="bsubsidy_total" >{{.BlockSummary.NextBlockSubsidy.Total}}</span></td>
                         </tr>
                     </tbody>
                 </table>
             </div>
         </div>
 
-        <h4><span>Ticket Fees</span></h4>
         <div class="row">
+            
             <div class="col-md-6">
-                <div>Tickets in Mempool: <strong id="mempoolfeeinfo_number">{{.MempoolFeeInfo.Number}}</strong></div>
-                <strong>Fees:</strong>
-                <table class="table table-sm">
-                    <thead>
-                        <th>Lowest Minable</th>
-                        <th>Min</th>
-                        <th>Max</th>
-                        <th>Median</th>
-                        <th>Mean</th>
-                    </thead>
-                    <tbody>
-                        <tr>
-                            <td><span id="mempoolfeeinfo_lowestmineable" >{{printf "%.8f" .MempoolFeeInfo.LowestMineable}}</span></td>
-                            <td><span id="mempoolfeeinfo_min" >{{printf "%.8f" .MempoolFeeInfo.Min}}</span></td>
-                            <td><span id="mempoolfeeinfo_max" >{{printf "%.8f" .MempoolFeeInfo.Max}}</span></td>
-                            <td><span id="mempoolfeeinfo_median" >{{printf "%.8f" .MempoolFeeInfo.Median}}</span></td>
-                            <td><span id="mempoolfeeinfo_mean" >{{printf "%.8f" .MempoolFeeInfo.Mean}}</span></td>
-                        </tr>
-                    </tbody>
-                </table>
+                <div><strong>Coin Supply:</strong> <span id="coin_supply" >{{.BlockSummary.CoinSupply}}</span></div>
```

Comment:
Swap "Coin Supply ..." and "Tickets in mempool..." so it better matches what's above them